src/attributes.jl

```diff
@@ -910,6 +910,29 @@ A model attribute for the number of results available.
 """
 struct ResultCount <: AbstractModelAttribute end
 
+"""
+    ConflictStatusCode
+
+An Enum of possible values for the `ConflictStatus` attribute. This attribute
+is meant to explain the reason why the conflict finder stopped executing in the
+most recent call to [`compute_conflict!`](@ref).
+
+Possible values are:
+* `COMPUTE_CONFLICT_NOT_CALLED`: the function [`compute_conflict!`](@ref) has
+  not yet been called
+* `NO_CONFLICT_FOUND`: there is no conflict because the problem is feasible
```

Comment:
It may also be that the IIS failed to find a conflict, despite the problem being infeasible.

Reply:
I'm adding a third status code (only available through Xpress) to catch that case. I guess most solvers will still use `NO_CONFLICT_FOUND`, though.